bindings/nodejs/lib/utils/utils.ts

```diff
@@ -324,4 +325,25 @@ export class Utils {
             data: { mnemonic },
         });
     }
+
+    /**
+     * Derives the `SlotCommitmentId` of the `SlotCommitment`.
+     */
+    static computeSlotCommitmentId(
```

Comment:
using a util function so we don't need to have bindings method calls in `types/`